rpcs3/Emu/Cell/PPUThread.cpp

```diff
@@ -951,6 +952,9 @@ void ppu_thread::cpu_task()
 				thread_ctrl::wait_on(g_fxo->get<rsx::thread>().is_inited, false);
 			}
 
+			g_progr = "Loading..."; // Keep this message simple
+			g_progr_show = true;
+
```

Comment:
I can probably remove this part and the progress_bar stuff.
I simply pushed the first build that worked